uk.co.compucorp.civicrm.hrleaveandabsences/templates/CRM/HRLeaveAndAbsences/Page/AbsenceTab.tpl

```diff
@@ -1,10 +1,53 @@
 <section data-leave-absences-absence-tab>
-  <ui-view></ui-view>
+  <div class="chr_absence-tab-page">
```

Comment:
`class="absence-tab-page"` keeping consistency which my leave and manager leave page.